### ISSUE TITLE
Error box upon creating new account iphone 5s 9.0.1

### ISSUE THREAD

**Issue by xpkoala** [Crypton/statusapp#143]
Upon creating a new account a white box appeared with the name 'undo' with two options for selection 'cancel' and 'undo'.

>Created account.
>Successfully logged in.
>Hit with 'undo' box
>Selected 'cancel'
>Have not had issues on account since.

Have been unable to reproduce upon making new accounts. Forgot to take a picture like an idiot.

**Comment by daviddahl** [comment]
this is a side effect of the safari browser engine and DOM. Not exactly sure. I think @devgeeks knows for sure and how to eliminate it

**Comment by devgeeks** [comment]
Didja SHAKE IT LIKE A POLAROID PICTURE?

otherwise I am not sure...

**Comment by daviddahl** [comment]
lol

**Comment by xpkoala** [comment]
Updated with picture. This occurred after uploading a picture to Timeline.

![image1](https://cloud.githubusercontent.com/assets/11012743/10474726/c5c1ad68-71ff-11e5-8625-1882fcaeb47d.PNG)

**Comment by daviddahl** [comment]
yeah not sure what this is all about. some wacked out webkit nonsense I bet
